source/fundamentals/monitoring/connection-monitoring.txt

```diff
@@ -29,8 +29,8 @@ application run faster.
 The following sections demonstrate how to record connection pool events in your
 application or want to explore the information provided in these events.
```

Comment:
S: out of scope for this ticket so not a necessary change, but this sentence is confusing. I think this wording makes more sense:
```suggestion
application and explore the information provided in these events.
```

Reply:
Good eye!